src/core/common/logging.hpp

```diff
@@ -53,12 +53,12 @@ extern "C" {
  *
  */
 #if OPENTHREAD_CONFIG_LOG_PREPEND_LEVEL
-#define _OT_LEVEL_NONE_PREFIX "[NONE]"
-#define _OT_LEVEL_CRIT_PREFIX "[CRIT]"
-#define _OT_LEVEL_WARN_PREFIX "[WARN]"
-#define _OT_LEVEL_NOTE_PREFIX "[NOTE]"
-#define _OT_LEVEL_INFO_PREFIX "[INFO]"
-#define _OT_LEVEL_DEBG_PREFIX "[DEBG]"
```

Comment:
Looking/thinking more about it, I think it may be better to keep the `[]` here and remove it from the function where we append the log string

These constants are used by both macro implementation and function one, so if we remove the `[ ]` here we need to update the macor implementation as well to add `[ ]`

Reply:
updated